### DISCUSSION TITLE
想问问成功排到2024bw的具体流程

### DISCUSSION THREAD

**Discussion by zunduwuyu** [mikumifa/biliTickerBuy#268] [提问专区]
我个人是用电脑端打开了购票页面（显示二维码那个界面）复制链接进入默认的间隔失败了，并且好像被封禁了20-30分钟？（就是去买其他不热门有很多余票的展子也显示前方拥挤，过了不知道多久又恢复正常了）。所以我分析我失败的原因会不会是链接的错误（就是我是不是该在手机端打开然后复制链接作为写入脚本的链接）还有间隔的原因，我是不是该调整为3000（300ms），希望有成功的小伙伴解答一下，感谢！！！！！！！！！！！！！！

**Comment by melon0918** [comment]
不是，你这就是单纯请求太快被风控了，没办法，多准备几个号和ip吧
